README.md

```diff
@@ -18,24 +18,58 @@ Your group will be both a "Customer" and a "Designer".  First you'll prepare for
 
 ***Q1.1 Briefly describe your Mission:***
 
+The mission is to create a concept where the object solves a task where a screen is usually required.
+
 ## Part 1.2: Finding a Designer/Customer (3 minutes)
 
 Find another group to work with. The other group should be students who neither of you have worked with before and far enough away that they didn't overhear any of your discussion of your Mission. One group will be "Designers" while another is "Customers".  Then the groups will switch roles (so each will be a Designer while the other is a Customer).  Pick which group is "Designing" first.
 
+We are designer first.
+
 ## Part 1.3 Empathizing (13 minutes total)
 
 (7 minutes) The Designer's first step in design thinking is empathizing with the person who the design is for.  The Designer should have a conversation with the Customer to get to know them and to ask about the Customer's needs and interests.  The goal is to try to identify the customer's need, *not a specific solution*.  The Designers who are going first should interview the customer and try to figure out the underlying need they'd like to satisfy.  
 
 ***Q1.2 Share some notes from your interview here and try to describe the "need" with a very brief description:***
 
+My problem: "How do you know when its going to rain?
+Isn't that annoying?
+Screen might not be good for you."
+
+Their problem:
+Wake up with a fever or hangover
+such a pain to get out of bed
+medicine far
+Don't want it on nightstand
+Does wear an apple watch
+
+Fine wearing things?
+
 (6 minutes) Reverse roles. The Customers are now Designers and vice versa.  Repeat the interview process. (Designers, complete the question above in that space)
 
 ## Part 1.4: Proposing solutions (8 minutes)
 
 Return to your original locations.  In this step all groups will act like Designers.  Another vital aspect of design thinking is open-minded brainstorming about potential solutions that will fulfill the Customer's needs.   Review your notes about the Customer's needs.  Use cards from the Things, Services, Human Actions, and Feedback decks to *design two (2) or more different solutions* to the Customer's needs.
 
+
+
+
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+Wearing something to bed that warns you that you're gonna be sick in the morning.
+
+Maybe temperature sensors in bed that detect fever, and deliver medicine to your bed.
+
+Delivery mechanism? medicine in air. delivery from afar. calls someone to bring you medicine. pills pop up from secret place that isn't annoying on the side of the bed.
```

Comment:
If medicine could be delivered via a mist (i.e. distributed by simply spraying it into the room in which someone is in), then this sort of treatment when someone is still in bed would be feasible.